criu/kerndat.c

```diff
@@ -1081,8 +1081,6 @@ int kerndat_init(void)
 		ret = kerndat_tun_netns();
 	if (!ret)
 		ret = kerndat_socket_unix_file();
-	if (!ret)
-		ret = kerndat_nsid();
```

Comment:
could you remove the next call. It is better to have kerndat_nsid and kerndat_link_nsid close to each other.

Reply:
Done.